olm-catalog/serverless-operator/Dockerfile

```diff
@@ -8,7 +8,7 @@ LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=serverless-operator
 LABEL operators.operatorframework.io.bundle.channel.default.v1="stable"
-LABEL operators.operatorframework.io.bundle.channels.v1="stable,stable-1.29,stable-1.30,stable-1.31,stable-1.32"
+LABEL operators.operatorframework.io.bundle.channels.v1="stable,stable-1.32"
```

Comment:
I believe all channels should be still listed in the annotation. In this case `stable,stable-1.32,stable-1.31,stable-1.30`.

I recall @Kaustubh-pande mentioned that when introducing the current channel format.

Reply:
From [docs](https://olm.operatorframework.io/docs/tasks/creating-operator-bundle/):
```
The label operators.operatorframework.io.bundle.channels.v1 reflects the list of channels the bundle is subscribing to when added into an operator registry
```
Listing all would mean that e.g. serverless 1.32 will be in channel stable-1.30, but the Index won't have it in metadata so it won't be installable anyway. 
I'd like to know the reason why to keep all versions here.

Reply:
When I look at productization files there are just two channels listed: https://pkgs.devel.redhat.com/cgit/containers/openshift-serverless-operator-bundle/tree/Dockerfile?h=openshift-serverless-1.31-rhel-8#n11

Reply:
@dsimansk we don't need all the channels there, it is something introducing a channel. The 1.32 release is part of stable and stable-1.32.

Reply:
Agreed, the description makes more sense then. I might have a bit blurred memory of the channels purpose. All good!